### ISSUE TITLE
Undead pets for Vampires

### ISSUE THREAD

**Issue by MiruNyashka** [TeamLapen/Vampirism#413]
It would be great if undead tamed animals like cats, dogs and horses could be controlled by vampire players. Also, vampires should have  a way to feed their pets manually so they don't have to waste time on hunting. For now we cannot ride a horse once it turns into a vampire version, also there's no way to tame a vampire wolf or ocelot or control their domestic versions.

**Comment by Webdragon444** [comment]
I agree, vampire pets would be cool. additionally, maybe a hunter could feed a wolf vampire blood to create hellhounds that sniff out vampires! there are so many possibilities.

**Comment by Octoham** [comment]
yes please

**Comment by AisuTistoto223** [comment]
I would also like to convert my pets into vampires.

**Comment by KGXSparks** [comment]
Would be dope
